features/networking/operator.feature

```diff
@@ -13,3 +13,39 @@ Feature: Operator related networking scenarios
     Then the expression should be true> cb.operator_status["status"]=="True"
     #Confirm whether network operator version matches with ocp version
     And the expression should be true> cluster_operator('network').version_exists?(version: cb.ocp_version)
+
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-22706
+  @admin
```

Comment:
this case is destructive, so we should add @destructive

Reply:
+1

Reply:
Agree. Thanks for pointing out that

Reply:
Done!